envs/monkey_zoo/packer/tasks/windows_disable_powershell_remoting.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Disable WinRM on startup
+  win_shell: |
+    Set-Service -Name "winrm" -StartupType Disabled
```

Comment:
Does this have side effects? More specifically, are there other protocols that run on top of WinRM that are now also disabled? It's fine if there are, as long as it's documented and doesn't cause tests to fail.

Reply:
Nope, it is not going to have any side effect. WinRM is used for PowerShell Remoting, WMI (not affected by it because WinRM is used in certain cases which we are not using them), Remote Server Managment Tools (not used) and probably that would be all.